docs/api.md

```diff
@@ -1,109 +1,44 @@
 
-Using the above-mentioned Swagger UI one can create valid `curl` requests using an interactive interface.
+### Parameters
 
-In the following sections examples for a GET and POST request for an Indicator of a region pre-defined by OQT are given.
+- `name`: Name of the Indicator
+- `layerName`: Name of the layer definition
+- `dataset`: Name of the dataset containing bounding polygons
+- `featureId`: Identifier of the feature in the dataset
+- `bpolys`: Bounding polygon(s) as GeoJSON Feature or FeatureCollection object. The Geometry of those objects has to be Polygon or MultiPolygon.
+- `layer`: Name, description and data of a custom layer. Data has to be provided in the same format as the response of the ohsome API would look like.
+- `includeSvg` (boolean): Include a SVG string of a figure displaying the Indicator
+results
+- `includeHtml` (boolean): Include a HTML string with the Indicator results
 
-### Request an Indicator for a pre-defined region
+Parameters are in part mutual exclusive (E.g. either `dataset` and `featureId` or `bpolys`. If the response contains multiple errors probably not all required parameters and/or mutual exclusive parameters has been provided.
 
-#### GET request
+### A Note on the Difference of `GET` and `POST` Requests
 
-```bash
-curl --request GET \
-    "https://oqt.ohsome.org/api/indicator?name=GhsPopComparisonBuildings&layerName=building_count&dataset=regions&featureId=3"
-```
-
-#### POST request
-
-```bash
-curl --request POST \
-    --header "Content-Type: application/json" \
-    --data '{
-        "name": "GhsPopComparisonBuildings",
-        "layerName": "building_count",
-        "dataset": "regions",
-        "featureId": 3
-    }' \
-    "https://oqt.ohsome.org/api/indicator"
-```
-
-> Tip: Use Python to pretty-print the response JSON -> `curl [...] | python3 -m json.tool`
-
-
-### Request an Indicator for custom bounding polygon(s)
-
-Since GeoJSON strings are quite big the `bpolys` parameter is not supported for `GET` requests.
-
-#### POST request
-
-Let's write down the request parameters in a `JSON` file.
-Note that the `bpolys` parameter needs to be a valid GeoJSON object.
-
-```bash
-{
-  "name": "GhsPopComparisonBuildings",
-  "layerName": "building_count",
-  "bpolys": {
-    "type": "Polygon",
-    "coordinates": [
-      [
-        [
-          8.674092292785645,
-          49.40427147224242
-        ],
-        [
-          8.695850372314453,
-          49.40427147224242
-        ],
-        [
-          8.695850372314453,
-          49.415552187316095
-        ],
-        [
-          8.674092292785645,
-          49.415552187316095
-        ],
-        [
-          8.674092292785645,
-          49.40427147224242
-        ]
-      ]
-    ]
-  }
-}
-```
+To the endpoint `/indicator` a request can be made using either the `GET` or `POST` method.
+For `GET` requests to either of those endpoints only the parameters `dataset` and `feature_id` are supported.
```

Comment:
is this referring to the indicators endpoint or also to other endpoints? here you write "those endpoints", but I'm not sure if there are other endpoints than `/indicator`

Reply:
Copy and paste error. I fixed this sentence.